src/Sulu/Bundle/ContentBundle/Tests/Unit/Content/Types/SmartContent/SmartContentTest.php

```diff
@@ -589,4 +573,51 @@ public function testGetViewDataPagedLimit($page, $pageSize, $offset, $limitResul
         $viewData = $this->smartContent->getViewData($property);
         $this->assertEquals(array_merge($config, array('page' => $page, 'hasNextPage' => $hasNextPage)), $viewData);
     }
+
+    private function getContentDataProperty()
+    {
+        $property = $this->getMockForAbstractClass(
```

Comment:
Definitely, that something I would say you can change as part of the boyscout rule ;-) But not really necessary in this PR.